js/components/file.js

```diff
@@ -68,6 +68,7 @@ Fliplet.FormBuilder.field('file', {
   },
   created: function() {
     Fliplet.FormBuilder.on('reset', this.onReset);
+    Fliplet.Hooks.on('beforeFormSubmit', this.onBeforeSubmit);
```

Comment:
Hi @squallstar, it will run for only file field and that to only before form is submitted because in created function we are only registering hook before form submit.

Reply:
@NikitaJadhavXoriant that's incorrect. Assume a scenario where multiple forms A, B and C are on a screen.

When form A is submitted, file fields for form B and C will run their `onBeforeSubmit` function.

Reply:
@squallstar, sorry I misunderstood, yes it will run for multiple form only for file field.

Reply:
Are you going to fix that then?

Reply:
@squallstar, yes fixed, please review.

cc @yogeshmsharma-xoriant

Reply:
@NikitaJadhavXoriant we already have internal features to tweak the data before it's submitted, why wouldn't we use this and make the relevant changes as necessary?

See https://github.com/Fliplet/fliplet-widget-form-builder/blob/dc0d55c77a60e889dfdd8111160b27afeb07590f/js/libs/form.js#L584-L612

Having a field itself responding to a system general hook is not a good design pattern. cc @tonytlwu

Reply:
@squallstar, done changes as per requested.

cc @yogeshmsharma-xoriant